.docsettings.yml

```diff
@@ -0,0 +1,10 @@
+# make a decision as to what sort of settings file should be used 
```

Comment:
Does it make to put this file somewhere other then the root directory here? I know when you get to .NET and Java I'm going to ask you to put it under the "eng" directory.

Reply:
Ok.

Reply:
Already supported and done in the Java repo.